guides/RestAPIGuide.md

```diff
@@ -1072,203 +1072,203 @@ The most common attachment type is 'file'.
+  "tags" : "<string>",
+  "duration" : "<int>",
   "links" : {
     "thumbnail" : ".../thumbs/[uuid]/[version]/[attach uuid]",
     "view" : ".../items/[uuid]/[version]/?attachment.uuid=[attach uuid]"
   }
 }
 ```
 **Flickr Attachment**
-```
+```json
 {
   * "type" : "flickr",
-  "uuid" : <uuid>,
-  * "description" : <string>,
-  "author" : <string>,
-  "datePosted" : <date>,
-  "dateTaken" : <date>,
-  "licenseName" : <string>,
-  "viewUrl" : <string>,
-  "thumbUrl" : <string>,
-  "licenseCode" : <string>,
-  "licenseKey" : <string>,
-  "photoId" : <string>,
-  "mediumUrl" : <string>,
+  "uuid" : "<uuid>",
+  * "description" : "<string>",
+  "author" : "<string>",
+  "datePosted" : "<date>",
+  "dateTaken" : "<date>",
+  "licenseName" : "<string>",
+  "viewUrl" : "<string>",
+  "thumbUrl" : "<string>",
+  "licenseCode" : "<string>",
+  "licenseKey" : "<string>",
+  "photoId" : "<string>",
+  "mediumUrl" : "<string>",
   "links" : {
     "thumbnail" : ".../thumbs/[uuid]/[version]/[attach uuid]",
     "view" : ".../items/[uuid]/[version]/?attachment.uuid=[attach uuid]"
   }
 }
 ```
 **iTunesU Attachment**
-```
+```json
 {
   * "type" : "itunesu",
-  "uuid" : <uuid>,
-  * "description" : <string>,
-  * "playUrl" : <string>,
-  "trackName" : <string>,
+  "uuid" : "<uuid>",
+  * "description" : "<string>",
+  * "playUrl" : "<string>",
+  "trackName" : "<string>",
   "links" : {
     "thumbnail" : ".../thumbs/[uuid]/[version]/[attach uuid]",
     "view" : ".../items/[uuid]/[version]/?attachment.uuid=[attach uuid]"
   }
 }
 ```
 **Package Resource Attachment**
-```
+```json
 {
   * "type" : "package-res",
-  "uuid" : <uuid>,
-  * "description" : <string>,
-  * "filename" : <string>,
-  "size" : <int>,
+  "uuid" : "<uuid>",
+  * "description" : "<string>",
+  * "filename" : "<string>",
+  "size" : "<int>",
   "links" : {
     "thumbnail" : ".../thumbs/[uuid]/[version]/[attach uuid]",
     "view" : ".../items/[uuid]/[version]/?attachment.uuid=[attach uuid]"
   }
 }
 ```
 **Content Package Attachment**
-```
+```json
 {
   * "type" : "package",
-  "uuid" : <uuid>,
-  * "description" : <string>,
-  "size" : <int>,
-  * "packageFile" : <string>,
+  "uuid" : "<uuid>",
+  * "description" : "<string>",
+  "size" : "<int>",
+  * "packageFile" : "<string>",
   "links" : {
     "thumbnail" : ".../thumbs/[uuid]/[version]/[attach uuid]",
     "view" : ".../items/[uuid]/[version]/?attachment.uuid=[attach uuid]"
   }
 }
 ```
 **Kaltura Attachment**
-```
+```json
 {
   * "type" : "kaltura",
-  "uuid" : <uuid>,
-  * "description" : <string>,
-  "mediaId" : <string>,
-  "title" : <string>,
-  "uploadedDate" : <date>,
-  "thumbUrl" : <string>,
-  "dataUrl" : <string>,
-  "tags" : <string>,
-  "duration" : <int>,
+  "uuid" : "<uuid>",
+  * "description" : "<string>",
+  "mediaId" : "<string>",
+  "title" : "<string>",
+  "uploadedDate" : "<date>",
+  "thumbUrl" : "<string>",
+  "dataUrl" : "<string>",
+  "tags" : "<string>",
+  "duration" : "<int>",
```

Comment:
Shouldn't need to quote integers.

Reply:
Okay, I'm stopping here (just saw how much more there is in this file). Please review the rest of the JSON below where you've added quotes to ensure only those needing quotes are quoted.

Reply:
https://github.com/equella/equella.github.io/pull/25#discussion_r234487858